prov/gni/include/gnix_util.h

```diff
@@ -108,6 +108,25 @@ extern atomic_t gnix_debug_next_tid;
 		abort(); \
 	} while (0)
 
+#if 1
```

Comment:
As a nit, I really dislike any if statement that is always true or false. If it is conditional, provide a condition. Otherwise it should be removed. 

Just my two cents...